resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/DriverConfigurationStepsOrchestrator.scala

```diff
@@ -52,6 +54,7 @@ private[spark] class DriverConfigurationStepsOrchestrator(
   private val filesDownloadPath = submissionSparkConf.get(INIT_CONTAINER_FILES_DOWNLOAD_LOCATION)
   private val dockerImagePullPolicy = submissionSparkConf.get(DOCKER_IMAGE_PULL_POLICY)
   private val initContainerConfigMapName = s"$kubernetesResourceNamePrefix-init-config"
+  private val hadoopConfigMapName = s"$kubernetesResourceNamePrefix-hadoop-config"
```

Comment:
Can we also name the auto-generated secret using kubernetesResourceNamePrefix and pass it down below so that secret is named after the job name? So that it is easier to find which secret is used by which spark job.

Reply:
We find secrets based on labels... secret name is irrelevant tho...

Reply:
although, you are right... this could be helpful. It will break my unit tests... but I guess it is worth for the sake of naming conventions :P

Reply:
The generated secret needs to have a unique name, and we've been using the kubernetes resource name prefix to guarantee uniqueness everywhere.